<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LockDialog</class>
 <widget class="QDialog" name="LockDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>272</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lock Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="meta_label">
     <property name="text">
      <string>This file is locked.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="edit_button">
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="lock_group_box">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Lock file</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="password_checkbox">
        <property name="text">
         <string>Change password</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="password">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="lock_settings">
        <property name="text">
         <string>Lock settings and models</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="lock_restrict">
        <property name="text">
         <string>Restrict to current narratives and resources</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="lock_all">
        <property name="text">
         <string>Lock all narratives and resources</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="lock_navigation">
        <property name="text">
         <string>Disable navigation</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="expiration_checkbox">
        <property name="text">
         <string>Model expires</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="expiration_label">
          <property name="text">
           <string>Expiration date</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="expiration_date"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="embed_models">
     <property name="text">
      <string>Embed all models</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Notes: Adding a password doesn't encrypt file contents. The password itself is encrypted. A file locked without a password cannot be unlocked.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="button_box">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
